A bug in GCJ's Java bytecode verifier causes compilation of je.jar
to native code to fail. Commenting out a debug trace call skips
over the problem area and should have no ill effects (unless you
want to trace that call!)

The bug: http://gcc.gnu.org/bugzilla/show_bug.cgi?id=19344

Apply this patch to the source, recompile to update Cleaner.class,
and replace it in the je.jar archive.

--- com-orig/sleepycat/je/cleaner/Cleaner.java	Thu Nov 18 17:48:26 2004
+++ com/sleepycat/je/cleaner/Cleaner.java	Sun Jan  9 02:13:19 2005
@@ -334,6 +334,7 @@
             } finally {
                 fileRetryInfo.endProcessing(canDeleteFile);
                 env.getFileManager().releaseExclusiveLock();
+                /*
                 Tracer.trace(Level.SEVERE, env,
                              "CleanerRun " + nCleanerRuns + 
                              " on file " + 
@@ -350,6 +351,7 @@
                              " nLNsMigrated=" + nLNsMigratedThisRun +
                              " nLNsLocked=" + nLNsLockedThisRun +
                              " nDeltasCleaned= " + nDeltasCleanedThisRun);
+                */
             }
         }
 
